src/System.Drawing.Common/src/System/Drawing/Printing/PrintingServices.Unix.cs

```diff
@@ -1105,42 +1057,60 @@ public struct CUPS_DESTS
 
         #endregion
 #pragma warning restore 649
-    }
-
-    class GlobalPrintingServicesUnix : GlobalPrintingServices
-    {
-        internal override PrinterSettings.StringCollection InstalledPrinters
+        internal static void LoadDefaultResolutions(PrinterSettings.PrinterResolutionCollection col)
         {
-            get
-            {
-                return PrintingServicesUnix.InstalledPrinters;
-            }
+            col.Add(new PrinterResolution(PrinterResolutionKind.High, (int)PrinterResolutionKind.High, -1));
+            col.Add(new PrinterResolution(PrinterResolutionKind.Medium, (int)PrinterResolutionKind.Medium, -1));
+            col.Add(new PrinterResolution(PrinterResolutionKind.Low, (int)PrinterResolutionKind.Low, -1));
+            col.Add(new PrinterResolution(PrinterResolutionKind.Draft, (int)PrinterResolutionKind.Draft, -1));
         }
+    }
 
-        internal override IntPtr CreateGraphicsContext(PrinterSettings settings, PageSettings default_page_settings)
+    internal class SysPrn
+    {
+        internal static void GetPrintDialogInfo(string printer, ref string port, ref string type, ref string status, ref string comment)
         {
-            return PrintingServicesUnix.CreateGraphicsContext(settings, default_page_settings);
+            PrintingServices.GetPrintDialogInfo(printer, ref port, ref type, ref status, ref comment);
         }
 
-        internal override bool StartDoc(GraphicsPrinter gr, string doc_name, string output_file)
+        internal class Printer
         {
-            return PrintingServicesUnix.StartDoc(gr, doc_name, output_file);
+            //public readonly string Name;
```

Comment:
nit: I guess we can remove this + the other commented out line alltogether?

Reply:
I cleaned up the file a bit more.